src/opm/input/eclipse/EclipseState/Tables/TableManager.cpp

```diff
@@ -123,36 +123,6 @@ std::optional<JFunc> make_jfunc(const Deck& deck) {
     return JFunc(deck);
 }
 
-DensityTable make_density_table(const GravityTable& gravity) {
```

Comment:
> Is moving this to the constructor needed or a matter of taste?

It's a little bit of both.  I've elected to go with construction from `initializer_list<RecordType>` instead of `vector<RecordType>` and there's no way to form an `initializer_list` at runtime.  That kind of forces this to go in a constructor.  That said, construction form `initializer_list<>` **is** a choice that's up for discussion.